nbs/tsdataset.ipynb

```diff
@@ -54,11 +54,12 @@
     "#| export\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #93.                for col in temporal_cols:
</pre><p>This could be done by making the temporal cols a set outside the loop <code>temporal_cols = set([target_col] + temporal_cols)</code> and then checking with <code>missing_cols = temporal_cols - col2pos.keys()</code>, which would allow us to display all the missing columns in the error message. Please also include the filename for easier debugging, e.g.</p>

<pre class="ql-syntax" spellcheck="false">if missing_cols:
    raise ValueError(f"Temporal columns: {missing_cols} not found in the file: {file}.")
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/1049/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
I left the target_col out of the set to make sure it is the first entry in the temporal_cols variable (which seems to be an assumption later on)